scripts/ci/kubernetes/minikube/start_minikube.sh

```diff
@@ -0,0 +1,81 @@
+#  Licensed to the Apache Software Foundation (ASF) under one   *
+#  or more contributor license agreements.  See the NOTICE file *
+#  distributed with this work for additional information        *
+#  regarding copyright ownership.  The ASF licenses this file   *
+#  to you under the Apache License, Version 2.0 (the            *
+#  "License"); you may not use this file except in compliance   *
+#  with the License.  You may obtain a copy of the License at   *
+#                                                               *
+#    http://www.apache.org/licenses/LICENSE-2.0                 *
+#                                                               *
+#  Unless required by applicable law or agreed to in writing,   *
+#  software distributed under the License is distributed on an  *
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+#  KIND, either express or implied.  See the License for the    *
+#  specific language governing permissions and limitations      *
+#  under the License.                                           *
+
+#!/usr/bin/env bash
+# Guard against a kubernetes cluster already being up
+kubectl get pods &> /dev/null
+if [ $? -eq 0 ]; then
+  echo "kubectl get pods returned 0 exit code, exiting early"
+  exit 0
+fi
+#
+
+curl -Lo minikube https://storage.googleapis.com/minikube/releases/v0.24.1/minikube-linux-amd64 && chmod +x minikube
```

Comment:
You probably want to store this in travis’ cache. See the Hadoop scripts as an example

Reply:
@dimberman sure will do.